src/panels/detail/DetailElement.tsx

```diff
@@ -389,59 +393,73 @@ export default class DetailElement extends React.Component<Props, State> {
                           value={this.state.inputTypeData}
                         />
                       </TableList>
-                      <h5>{Locale[AppSettings.viewLanguage].intrinsicTropes}</h5>
-                    <IntrinsicTropeTable
-                      iri={WorkspaceElements[this.state.id].iri}
-                      tropes={getIntrinsicTropeTypes(this.state.id)}
-                      onEdit={(id: string) => this.props.updateDetailPanel(id)}
-                      onRemove={(id: string) => {
-                        const conn = Object.keys(WorkspaceLinks).find(
-                          (link) =>
-                            WorkspaceLinks[link].active &&
-                            ((WorkspaceLinks[link].iri ===
-                              parsePrefix("z-sgov-pojem", "má-vlastnost") &&
-                              WorkspaceLinks[link].source === this.state.id &&
-                              WorkspaceLinks[link].target === id) ||
-                              (WorkspaceLinks[link].iri ===
-                                parsePrefix("z-sgov-pojem", "je-vlastností") &&
-                                WorkspaceLinks[link].source === id &&
-                                WorkspaceLinks[link].target === this.state.id))
-                        );
-                        if (conn)
+                      <h5>
+                        {Locale[AppSettings.viewLanguage].intrinsicTropes}
+                      </h5>
+                      <IntrinsicTropeTable
+                        iri={WorkspaceElements[this.state.id].iri}
+                        tropes={getIntrinsicTropeTypes(this.state.id)}
+                        onEdit={(id: string) =>
+                          this.props.updateDetailPanel(id)
+                        }
+                        onRemove={(id: string) => {
+                          const conn = Object.keys(WorkspaceLinks).find(
+                            (link) =>
+                              WorkspaceLinks[link].active &&
```

Comment:
This code is not convenient to read at all. Too much nesting, too complex conditions, no abstractions whatsoever...